galsim/roman/roman_bandpass.py

```diff
@@ -134,8 +139,7 @@ def getBandpasses(AB_zeropoint=True, default_thin_trunc=True, **kwargs):
     bandpass_dict = {}
     # Loop over the bands.
     for index, bp_name in enumerate(data.dtype.names[1:]):
-        # Need to skip the prism and grism (not used for weak lensing imaging).
-        if bp_name=='SNPrism' or bp_name=='Grism_1stOrder' or bp_name=='Grism_0thOrder':
+        if include_all_bands is False and bp_name in ('Grism_0thOrder', 'Grism_1stOrder', 'SNPrism'):
```

Comment:
Do we want to save this list somewhere (and reuse them in the tests may be) instead of hardcoding them here?

Reply:
I could imagine this going in `galsim/roman/__init__.py` as `nonimaging_bands` or something.

Reply:
Sure.